### PR TITLE
Product Based Navigation

### PR DESCRIPTION
**Pull request by revomhere**

### CODE REVIEW

**Comment by ihordiachenko** [comment]
image

Remove nesting in "Overview", leave just the /overview page

**Comment by ihordiachenko** [comment]
`worldcoin-state-replication/overview` - wrong document used. Leave overview blank and move the contents to the replication guide page

**Comment by ihordiachenko** [comment]
`products/rarimo-protocol/overview` - leave this one blank and move contents to "Rarimocore Module" page

**Comment by ihordiachenko** [comment]
`products/rarimo-protocol/api-reference` - remove Subgraph section completely, move the JS SDK stuff to bridging protocol

File: docs/products/proof-of-humanity/guides/using-rarimos-proof-of-humanity-plugin-for-galxe.mdx [hunk 1]
```diff
@@ -80,20 +77,20 @@ To add Proof of Human check, set up your campaign using the instructions below.
...
 
 7. Save the Credential Group:
 
-   ![Save the Group](/img/galxe-step-7.png)
+   ![Galxe_Step_6.png](/img/guide-galxe/Galxe_Step_6.png)
 
 8. After you are done with the rest of the details for your campaign, you should see the Proof of Human step on the campaign page:
 
-   ![Proof of Human step should be visible now](/img/galxe-step-8.png)
+   ![Galxe_Step_7.png](/img/guide-galxe/Galxe_Step_7.png)
```
Comment (ardier16) [inline docs/products/proof-of-humanity/guides/using-rarimos-proof-of-humanity-plugin-for-galxe.mdx:96]: same for labels

**Review by ardier16 — APPROVED** [review]

**Comment by ihordiachenko** [comment]
@revomhere @ardier16 I think we should reduce the level of nesting, kinda like this:

Screenshot 2024-01-18 at 20 54 28

**Products** is a non-clickable label

**Comment by ihordiachenko** [comment]
@revomhere 
1. Make the "Products" label visually distinct from the links and remove hover effect
2. Remove the left margin
Screenshot 2024-01-19 at 13 18 12
 on product items

**Review by ihordiachenko — APPROVED** [review]

**Pull request merged**